### ISSUE TITLE
Installing?

### ISSUE THREAD

**Issue by Nixontira** [nitaybz/homebridge-broadlink-mp#3]
Hey,
im trying to install this specific fork on my homebridge, but it always installs the one by smka when i type the installcommand. But that version doesnt work with my MP1 it jsut crashes my Homebridge.

Just saw @nitaybz you updated this fork  just a bit ago. But how do i install this exact fork? Sorry for this nooby question, quite new in homebridge universe.

**Comment by nitaybz** [comment]
`sudo npm install -g https://github.com/nitaybz/homebridge-broadlink-platform`

this will install my version from github

**Comment by Nixontira** [comment]
Thanks! Fingers crossed this will help.

**Comment by Nixontira** [comment]
hmmm I get the following error when trying to install :/

`
pi@hbridge:~ $ sudo npm install -g https://github.com/nitaybz/homebridge-broadlink-platform
npm ERR! code 128
npm ERR! Command failed: /usr/bin/git clone -q https://github.com/nitaybz/homebridge-broadlink-platform.git /root/.npm/_cacache/tmp/git-clone-eef9073a
npm ERR! fatal: Konnte führende Verzeichnisse von '/root/.npm/_cacache/tmp/git-clone-eef9073a' nicht erstellen.: Keine Berechtigung
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-03-27T18_13_05_625Z-debug.log
`

**Comment by nitaybz** [comment]
This is the new npm issue with installation from github. Try to install older version of npm:
`sudo npm install -g npm@4.6.1`

Then reinstall like I showed you before

**Comment by Nixontira** [comment]
im getting closer! your trick worked for me. Plugin is installed and the four sockets show up on my device. But it doesn't work. Terminal log says that the Homebridge is searching for mp1 device. but on the other side, Homebridge says in the beginning it found a MP1 on ip xxxxxx and Mac xxxxxx. another Problem is, that now no other plugin works anymore. 

one thing I noticed is, that in your sample config, there is no ip for mp1 just for the sp modules? that right? all the other forks asked for an specific ip.

edit:

okay I added the ip adress and now everything works flawlessly ! 

**thanks man !**
